post.md

```diff
@@ -325,50 +325,50 @@ It would get called if the component reanimated successfully then unmounted, but
 For example if `Albums` started to render, made the promise and suspended. Then something in the application changed before the promise resolved, maybe the user navigates away or decides on a different band to look at. `Albums` is never reanimated by suspense, so we would never get that `useEffect` cleanup.
```

Comment:
sorry, nearest place I could comment. 

I think a lovely flow diagram wouldn't go a miss above. 

What do you mean "finally" at something tricky? You've already said there's pitfalls aplenty. 
"Thar be dragons?" - so many references and sayings in this inner monologue 😆 
I missed a capital "D" for line 319

I want some simple cause and effect scenarios here (with either diagrams or super simple) as I'm a little lost by the end of this paragraph. We (the user) have wandered off and left Albums, the promise is still hanging round making leaks, right?